src/System.Private.Xml.Linq/tests/events/System.Xml.Linq.Events.Tests.csproj

```diff
@@ -30,6 +30,7 @@
       <Project>{52666206-b6c9-49fa-a1d7-d0a0c68807b0}</Project>
       <Name>XDocument.Common</Name>
     </ProjectReference>
+    <ProjectReference Include="..\XDocument.Test.ModuleCore\XDocument.Test.ModuleCore.csproj" />
```

Comment:
Should we make this reference uwp specific? also, we might need to add the `KeepProjectReference` metadata or else I'm not sure how Helix will resolve this as there is no package for that project.